torch/_tensor_docs.py

```diff
@@ -1768,14 +1768,14 @@ def add_docstr_all(method, docstr):
 
 add_docstr_all('index_add_',
                r"""
-index_add_(dim, index, tensor, *, alpha=1) -> Tensor
+index_add_(dim, index, input, *, alpha=1) -> Tensor
```

Comment:
The actual name of this argument is `index`, according to https://github.com/pytorch/pytorch/blob/7191dd2613051b1ddfe7f5a9cfd0397b1444a116/aten/src/ATen/native/native_functions.yaml#L5650